install.html

```diff
@@ -202,6 +213,50 @@ <h4>7. Other INI files</h4>
 
       </ul>
 
+      <h2>2. Install Cypht within Tiki</h2>
+      <p>If you have tiki installed, you can use Cypht within tiki. This is an easy way of installing Cypht.<br/>
+        Please follow the following link for a complete guide of how to install and use cypht within Tiki.</br>
+        <a href="https://doc.tiki.org/Webmail">https://doc.tiki.org/Webmail</a>
+      </p>
+
+      <h2>3. Install cypht using Docker</h2>
+      <p>
+        Using Docker is one of the easiest way to install cypht as the cypht docker image comes with all the steps required in the manual instalation done for you. But the bad news is that this instllation way requires docker knowledge.<br/>
+        Here is the cypht docker repository: <a href="https://hub.docker.com/r/sailfrog/cypht-docker">https://hub.docker.com/r/sailfrog/cypht-docker</a><br/>
+        To run containers required by cypht, please, first make sure you have docker and docker-compose installed on your system, then take a look on the section "exemple docker-compose" of repository overview, then do the following: 
+        <ul>
```

Comment:
@rodriguezny I think it is an error to put ul tag inside a p tag

Reply:
Yes, it is not considered as valid html, I fixed it. Thanks!